docs/core/system-prompt.md

```diff
@@ -0,0 +1,77 @@
+# System Prompt Override (GEMINI_SYSTEM_MD)
+
+The core system instructions that guide Gemini CLI can be completely replaced with your own Markdown file. This feature is controlled via the `GEMINI_SYSTEM_MD` environment variable.
+
+## Overview
+
+- `GEMINI_SYSTEM_MD` tells the CLI to use an external Markdown file as the system prompt instead of the built‑in default.
```

Comment:
Suggest **one** of the following:

- Removing "## Overview" and segueing into the bullet list. 
- Turning the bullet list into a paragraph.
- Adding a sentence to introduce the bullet list.